docs/advanced_topics/api/v2/configuration.md

```diff
@@ -0,0 +1,267 @@
+# Wagtail API v2 Configuration Guide
+
+This section of the docs will show you how to set up a public API for your
+Wagtail site.
+
+Even though the API is built on Django REST Framework, you do not need to
+install this manually as it is already a dependency of Wagtail.
+
+## Basic configuration
+
+### Enable the app
+
+Firstly, you need to enable Wagtail's API app so Django can see it.
+Add `wagtail.api.v2` to `INSTALLED_APPS` in your Django project settings:
+
+```python
+# settings.py
+
+INSTALLED_APPS = [
+    ...
+
+    'wagtail.api.v2',
+
+    ...
+]
+```
+
+Optionally, you may also want to add `rest_framework` to `INSTALLED_APPS`.
+This would make the API browsable when viewed from a web browser but is not
+required for basic JSON-formatted output.
+
+### Configure endpoints
+
+Next, it's time to configure which content will be exposed on the API. Each
+content type (such as pages, images and documents) has its own endpoint.
+Endpoints are combined by a router, which provides the url configuration you
+can hook into the rest of your project.
+
+Wagtail provides three endpoint classes you can use:
+
+-   Pages [:class:\`wagtail.api.v2.views.PagesAPIViewSet\`]()
+-   Images [:class:\`wagtail.images.api.v2.views.ImagesAPIViewSet\`]()
+-   Documents [:class:\`wagtail.documents.api.v2.views.DocumentsAPIViewSet\`]()
```

Comment:
This doesn't quite look right.

@lb- what would you recommend for this type of reference? Can't see anything in the guidelines.

note that the only usage I could find of `:class:` in our markdown docs are in `{eval-rst}` blocks - https://cs.github.com/?scopeName=All+repos&scope=&q=repo%3Awagtail%2Fwagtail+%3Aclass%3A+path%3A**%2F**.md

Reply:
Yeah. Best to use the eval-rst for class autodoc

Reply:
eval-rst used to match the suggestion